tests/norm/test_fused_dit_layernorm.py

```
@@ -0,0 +1,737 @@
+    device = torch.device("cuda")
+    torch.manual_seed(42)
+
+    batch_size, seq_len = 1, 768
+    input_tensor = torch.randn(
+        batch_size, seq_len, HIDDEN_DIM, dtype=torch.bfloat16, device=device
+    )
+    temb_data = _make_wan_temb_inputs(batch_size, seq_len, HIDDEN_DIM, device)
+    temb_chunks = temb_data["temb"].chunk(6, dim=2)
+    table_chunks = temb_data["scale_shift_table"].chunk(6, dim=1)
+
+    residual_fused, norm_fused = fused_dit_residual_layernorm_scale_shift(
```

Ruff (0.15.11)

[warning] Unpacked variable `residual_fused` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)